schema/rockcraft.json

```diff
@@ -3795,6 +3806,7 @@
                   "meson",
                   "nil",
                   "npm",
+                  "poetry",
```

Comment:
Since we're disabling poetry do we want to temporarily remove it from the JSON? Or is that too much work for something we're about to re-enable?

Reply:
it's not disabled on rockcraft, so adding it to the schema here seems 'less wrong' heh

Reply:
Ohh, I thought it was. My bad. Thanks!